Forge/src/main/java/me/ferdz/placeableitems/tileentity/StackHolderTileEntity.java

```diff
@@ -1,24 +1,23 @@
 package me.ferdz.placeableitems.tileentity;
 
-import me.ferdz.placeableitems.init.PlaceableItemsBlockRegistry;
+import me.ferdz.placeableitems.PlaceableItems;
 import net.minecraft.item.ItemStack;
 import net.minecraft.nbt.CompoundNBT;
-import net.minecraft.network.NetworkManager;
-import net.minecraft.network.play.server.SUpdateTileEntityPacket;
 import net.minecraft.tileentity.TileEntity;
 import net.minecraft.tileentity.TileEntityType;
-import net.minecraftforge.items.ItemStackHandler;
-
-import javax.annotation.Nullable;
+import net.minecraftforge.registries.ObjectHolder;
 
 public class StackHolderTileEntity extends TileEntity {
 
+    @ObjectHolder(PlaceableItems.MODID + ":writable_book_block")
```

Comment:
Could you explain what this does?

Reply:
This injects the registered instance into the field. A feature added in Forge 1.13. The annotation can be applied to a class (in which all fields are injected), or onto specific fields.
https://mcforge.readthedocs.io/en/1.15.x/concepts/registries/